.github/workflows/docs-build.yml

```diff
@@ -0,0 +1,17 @@
+name: docs-build
+
```

Comment:
@froschdesign can you check if this workflow is OK as-is, or if more is needed?

Reply:
@Ocramius 
Looks good! 👍 

_Sorry for the delay, too many notifications here on GitHub._